Makefile

```diff
@@ -23,9 +23,9 @@ update-package-locks:
 	# https://npm.community/t/some-packages-have-dist-tarball-as-http-and-not-https/285/13
 	rm -rf services/node_modules/ services/package-lock.json && \
 	rm -rf distribution/client/node_modules/ distribution/client/package-lock.json && \
-	cd services/ &&	$(NODE) npm install && \
+	cd services/ &&	$(NODE) npm ci && \
```

Comment:
I think :bug:.
This one is actually designed to touch/update the `package-lock.json` files. So, AIU the `npm ci` task, I suspect this would just fail because there's *no* package-lock file when it's called.

Going to check locally.

Reply:
Confirmed

```
$ make update-package-locks                                                                                                                                                                                                       d04b1ce
# Sigh, the sed for forcing https below is recommended by the npm registry team itself...
# https://npm.community/t/some-packages-have-dist-tarball-as-http-and-not-https/285/13
rm -rf services/node_modules/ services/package-lock.json && \
rm -rf distribution/client/node_modules/ distribution/client/package-lock.json && \
cd services/ &&  npm ci && \
sed -i 's/"resolved": "http:/"resolved": "https:/g' package-lock.json && \
cd ../distribution/client &&  npm ci && \
sed -i 's/"resolved": "http:/"resolved": "https:/g' package-lock.json
npm ERR! cipm can only install packages with an existing package-lock.json or npm-shrinkwrap.json with lockfileVersion >= 1. Run an install with npm@5 or later to generate it, then try again.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/tiste/.npm/_logs/2018-10-25T09_11_12_808Z-debug.log
make: *** [Makefile:24: update-package-locks] Error 1
```

Reply:
Fixed it myself with last commit.